viewing_party/party.py

```diff
@@ -1,23 +1,140 @@
+
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        new_movie = {
+            "title": title,
+            "genre": genre,
+            "rating": rating,
+        }
+        return new_movie

```

Comment:
When you're immediately returning a value you don't need to store it in a variable:

```suggestion
        return {
            "title": title,
            "genre": genre,
            "rating": rating,
        }
```